src/components/board/Profile.js

```diff
@@ -1,7 +1,50 @@
 import React from "react";
+import Image from "next/image";
+import { MdEmail } from "react-icons/md";
+import { GrLinkedinOption } from "react-icons/gr";
+import { IoMdStar } from "react-icons/io";
+import Link from "next/link";
 
-const profile = () => {
-  return <div>profile</div>;
+const Profile = ({
+  name,
+  image,
+  title,
+  email,
+  linkedin,
+  year,
+  major,
+  pronouns,
+}) => {
+  return (
+    <div className="border-2 border-black font-poppins bg-white w-full h-auto rounded-xl">
```

Comment:
fix fonts styles and sizes based on Figma. Check figma in dev mode to see the details